apps/aesophia/Makefile

```diff
@@ -0,0 +1,2 @@
+compile_test_aes:
+	( cd scripts && ./compile_test_contract $(CONTRACT) ../../../_build/default/lib/ ;)
```

Comment:
Some rebar3 escript config may prevent using it this way and actually enable user to use script from released binary package. Though as this is only for easing dev testing ok.